sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1405,6 +1450,16 @@ class HistGradientBoostingClassifier(ClassifierMixin, BaseHistGradientBoosting):
 
         .. versionadded:: 0.23
 
+    interaction_cst : list of lists or sets of int, default=None
+        Each inner list or set specifies the feature indices that are allowed
+        to interact with each other, meaning splits in child nodes are only
+        allowed in those sets. For instance, `[{0, 1}, {2, 3, 4}]` says that
+        a single tree is either only split on features 0 and 1 or only on
+        features 2, 3 and 4. If there are more featres, e.g. 5 and 6, those
```

Comment:
```suggestion
        features 2, 3 and 4. If there are more features, e.g. 5 and 6, those
```

Reply:
I'll change this behaviour, i.e. not specified features will automatically go in as one group.